config/s390x/bli_family_s390x.h

```diff
@@ -0,0 +1 @@
+#define BLIS_SIMD_SIZE 16 // 128-bit SIMD
```

Comment:
@fgvanzee's gonna want the whole copyright header in here, maybe some additional boilerplate.

Reply:
Yes, license header please kthnx.